### PR TITLE
[Fix] Cell's output with no data

### PR DESCRIPTION
**Pull request by TanguyFi**
**Cell's output with no data**

A notebook with a cell containing a `print` instruction fails to export because the cell's output does not contain `data` key. This PR fix it.

@m-rossi Thank you for this useful repo ! 🙏   Open to discuss about this 🙂 

**Checklist**

- [x] Tests have passed
- [x] New code is covered with tests

### CODE REVIEW

**Comment by m-rossi** [comment]
@TanguyFi thanks for your contribution. I was not aware of this issue.

Don't mind the failing tests, they do sometimes. I was unable to track down why. I restarted CI.

**Comment by TanguyFi** [comment]
@m-rossi Tested on my environment, it's ok ! 👌  Could you merge this and do a new release ? 🙂

**Comment by m-rossi** [comment]
Please add the test I described above. If that's in I will happily merge.

**Review by m-rossi — CHANGES_REQUESTED** [review]

**Comment by m-rossi** [comment]
Looks like I have forgotten to press submit, until now it was only visible to me.

**Comment by TanguyFi** [comment]
Seems good ? 🙂

**Comment by m-rossi** [comment]
In general: I already have a fixture with simple print statements: https://github.com/m-rossi/jupyter-docx-bundler/blob/master/jupyter_docx_bundler/tests/conftest.py#L227-L252 The related test also checks if the output is kept although the input is hided.

Is your error may a combination of print statements with pandas-html-tables?

**Comment by TanguyFi** [comment]
Yes, exactly but more generally  when a cell implies a `print` but a return an output with `data` key.

**Comment by m-rossi** [comment]
Ok, understood. This must be related to code I introduced to import pandas-html-tables.

In the fix you created is the pandas-table a nice world table after exporting?

**Comment by TanguyFi** [comment]
I think everything is ok: 

image

File: jupyter_docx_bundler/tests/conftest.py [hunk 1]
```diff
@@ -499,20 +499,46 @@ def math_notebook(request):
     return request.param
 
 
+@pytest.fixture()
```
Comment (m-rossi) [inline jupyter_docx_bundler/tests/conftest.py:502]: We do not need an extra test for this error. Please remove.

File: jupyter_docx_bundler/converters.py [hunk 2]
```diff
@@ -170,7 +170,8 @@ def preprocess(content, path, handler=None):
         # process outputs
         if 'outputs' in cell and any(['data' in output for output in cell['outputs']]):
```
Comment (m-rossi) [inline jupyter_docx_bundler/converters.py:171]: Please remove the additional check for data in this if-clause. We do this better in your code below.

File: jupyter_docx_bundler/tests/test_converters.py [hunk 3]
```diff
@@ -227,3 +227,31 @@ def test_pandas_html_table(tmpdir, pandas_html_table_notebook):
     assert all(df_md.index == df.index), 'Index does not match'
     assert all(df_md.columns == df.columns), 'Columns does not match'
     np.testing.assert_allclose(df_md.values, df.values, atol=1e-5)
+
+
+def test_print_with_html(tmpdir, print_with_html_notebook):
```
Comment (m-rossi) [inline jupyter_docx_bundler/tests/test_converters.py:232]: We do not need an extra fixture. Please remove.

**Review by m-rossi — CHANGES_REQUESTED** [review]

**Review by m-rossi — APPROVED** [review]

**Comment by m-rossi** [comment]
Thanks @TanguyFi for this bugfix, will merge and release in the next hours.

**Pull request merged**